src/main/java/me/crespel/runtastic/RuntasticExportConverter.java

```diff
@@ -61,7 +69,25 @@ protected void doList(File path) throws FileNotFoundException, IOException {
 		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
 		List<SportSession> sessions = converter.listSportSessions(path);
 		for (SportSession session : sessions) {
-			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", type: " + session.getSportTypeId() + ", duration: " + session.getDuration() + "s");
+			System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() + ", Sport Type: " + session.getSportTypeId() + ", duration: " + session.getDuration()/60000 + "min");
+		}
+	}
+
+	protected void doInfo(File path, String id) throws FileNotFoundException, IOException {
+		SimpleDateFormat sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
+		List<SportSession> sessions = converter.listSportSessions(path);
+		for (SportSession session : sessions) {
+			if ( session.getId().equalsIgnoreCase(id) ) {
+				System.out.println(sdf.format(session.getStartTime()) + " - ID: " + session.getId() );
+				System.out.println("      Sport Type: " + session.getSportTypeId() + ", Surface Type: " + session.getSurfaceId() + ", Feeling Id: " + session.getSubjectiveFeelingId());
+				System.out.println("      Duration: " + session.getDuration() + " ms / " + session.getDuration()/1000 + " s / " + session.getDuration()/60000 + " min");
+				System.out.println("      Distance: " + session.getDistance().floatValue()/1000 + " km, Calories: " + session.getCalories());
+				System.out.println("      Avg Pace: " + session.getDurationPerKm().floatValue()/60000 + " min/km");

```

Comment:
Instead of calling `.floatValue()` on these Integers, you could just divide them by 1000.0 (i.e. a double) or 60000.0.

But careful! some sessions may have no distance (no GPS) or duration per km, so these two lines can throw a NullPointerException at runtime. This can be fixed by checking if the value is not null before displaying the line.

Reply:
Will be fixed in next commit.